proposals/pubsub.md

```diff
@@ -0,0 +1,182 @@
+- Enable / upgrade IPFS nodes operated by PL.
+- Raise awareness (blog, tweet, support community transition)
+
+#### What does done look like?
+_What specific deliverables should completed to consider this project done?_
+
+- Shipped release of go-ipfs with pubsub enabled
+- Increased number of pubsub enabled IPFS nodes in the network.
+
+####  What does success look like?
+_Success means impact. How will we know we did the right thing?_
+
+- Various projects (like OrbitDB) no longer have to explain with instructions how to enable pubsub.
+- Teams like fission deliver pubsub to their users requesting it.
+- Pubsub becomes more reliable due to increased number of enabled nodes. 
+
+
+<!--
+Provide success criteria. These might include particular metrics, desired changes in the types of bug reports being filed, desired changes in qualitative user feedback (measured via surveys, etc), etc.
+-->
+
+#### Counterpoints &amp; pre-mortem
+_Why might this project be lower impact than expected? How could this project fail to complete, or fail to be successful?_
+
+- Everyone enables pubsub already anyway.
+- Pusub prooves (more) unreliable even with increased number of enabled nodes.
+
+#### Alternatives
+_How might this project’s intent be realized in other ways (other than this project proposal)? What other potential solutions can address the same need ?_
+
+- Alternative messaging primitives (e.g. Federated SSB like pubs)
+- [SMTP][] support in IPFS nodes (you can email any node by email address)
+
+#### Dependencies/prerequisites
+<!--List any other projects that are dependencies/prerequisites for this project that is being pitched.-->
+
+- none
+
+#### Future opportunities
+<!--What future projects/opportunities could this project enable?-->
+
+- Improving IPNS (through IPNS over pubsub)
+- DNS Service Discovery (DNS-SD) but over the global IPFS network through PubSub.
+
+## Required resources
+
+#### Effort estimate
+<!--T-shirt size rating of the size of the project. If the project might require external collaborators/teams, please note in the roles/skills section below). 
+For a team of 3-5 people with the appropriate skills:
+- Small, 1-2 weeks
+- Medium, 3-5 weeks
+- Large, 6-10 weeks
+- XLarge, >10 weeks
+Describe any choices and uncertainty in this scope estimate. (E.g. Uncertainty in the scope until design work is complete, low uncertainty in execution thereafter.)
+-->
+
+- Small to Medium
+
+> I am not really qualified to answer this question. If we find overhead reasonable, enabling it should be metter of switching defaults.
+
+#### Roles / skills needed
+<!--Describe the knowledge/skill-sets and team that are needed for this project (e.g. PM, docs, protocol or library expertise, design expertise, etc.). If this project could be externalized to the community or a team outside PL's direct employment, please note that here.-->
+
+- go-ipfs contributor
```

Comment:
Some amount of go-ipfs knowledge might be needed to actually flip the "on" switch for making pubsub enabled by default, but basically everything here is in pubsub.

The ideal candidate would be someone who has some experience with go-libp2p-pubsub, but anyone whose spend some time with go-libp2p could do this. A Go engineer who hasn't worked with go-libp2p yet would likely be fine here too, but would likely need some guidance.

Reply:
@aschmahmann should I change to ?

```suggestion
- go-libp2p contributor (or go-ipfs contributor with some guidance) 
```

Reply:
yep, lgtm